impl/src/main/java/com/okta/jwt/impl/jjwt/TokenVerifierSupport.java

```diff
@@ -63,6 +65,13 @@ protected Jwt decode(String token, JwtParser parser, ClaimsValidator claimsValid
                     jwt.getBody().getIssuedAt().toInstant(),
                     jwt.getBody().getExpiration().toInstant(),
                     jwt.getBody());
+        } catch (IncorrectClaimException e) {
+            // add a possible cause if the issuer is invalid
+            if (Claims.ISSUER.equals(e.getClaimName())) {
```

Comment:
Might this better be managed with a cause rather than needing one of more specific `if` statements? If you are going to keep it, you need a space between the end `"` and the `+` below.